spec.md

```diff
@@ -725,7 +725,13 @@ Plugins MAY create 3 types of volumes:
 
 - Empty volumes. When plugin supports `CREATE_DELETE_VOLUME` OPTIONAL capability.
 - From an existing snapshot. When plugin supports `CREATE_DELETE_VOLUME` and `CREATE_DELETE_SNAPSHOT` OPTIONAL capabilities.
+  The Plugin SHOULD create a volume that looks like exact copy of the original snapshotted volume at the time the snapshot was taken.
```

Comment:
If feels to me like we could de-duplicate some of the language here and below, by including the requirements about resizing the filesystem in a later paragraph and referring to that twice in the list.

Also, why are we hedging with SHOULD? It feels like MUST would be appropriate here.

Reply:
Replaced with MUST and moved to a separate paragraph below.

@bswartz PTAL